src/components/ModalPurchaseHbar.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <Modal
+        :is-open="isOpen"
+        hide-decoration
+        @change="this.$listeners.change"
+    >
+        <iframe
+            id="carbonFiberIFrameExample"
+            title="Carbon Fiber iframe Example"
+            width="450"
+            height="800"
+            allow="fullscreen"
+            :src="url"
+        />
+    </Modal>
+</template>
+
+<script lang="ts">
+import Modal from "../components/Modal.vue";
+import {
+    createComponent,
+    PropType,
+    computed
+} from "@vue/composition-api";
+import { store } from "../store";
+
+export default createComponent({
+    components: { Modal },
+    model: {
+        prop: "isOpen",
+        event: "change"
+    },
+    props: { isOpen: (Boolean as unknown) as PropType<boolean> },
+    setup() {
+        const accId = store.state.wallet.session!.account;
+        const accIdString = accId.toString();
+        const environment = process.env.NODE_ENV !== "production" ? "sandbox" : "production";
+        const key = process.env.NODE_ENV !== "production" ? "89fa28dd-b26e-4af4-8313-1536054767d5" : "production";
```

Comment:
huh?

The key (promo code) is `89...` for non-production and `production` for production?

Reply:
Just a placeholder for now